Task_10/style1.css

```diff
@@ -0,0 +1,551 @@
+  }
+  
+  .menu__box {
+    display: block;
+    position: fixed;
+    visibility: hidden;
+    top: 0;
+    left: -100%;
+  
+    width: 300px;
+    height: 100%;
+  
+    margin: 0;
+    padding: 80px 0;
+  
+    list-style: none;
+  
+    background-color:  #f39509e3;
+    box-shadow: 1px 0px 6px rgba(0, 0, 0, .2);
+  
+    transition-duration: .25s;
+  }
+  
+  .menu__item {
+    display: block;
+    padding: 12px 24px;
+  
+    color: #333;
+  
+    font-family: BebasNeueRegular; 
+    font-size: 30px;
+    font-weight: 400;
+  
+    text-decoration: none;
+  
+    transition-duration: .25s;
+  }
+  .menu__item:hover {
+    background-color: #CFD8DC;
+  }
+
+/* ///////////////////////// */}
+    
+.orange{
+    color: #FBA929;
+    letter-spacing: 1px;
+    padding-left: 8px;
+    
+}
+
+.rabbit_text4{
+    color: rgba(59, 59, 59, 0.938);
+
+}
+/* ////////////////// */
+.container{
+    display:flex;
+    border: solid#FBA929  ;
+    width: auto;
+    height: 400px; 
+    background-image: url(./img/big-orange-bg.png);
+    margin-top: 1%;
+    background-size: cover;
+    background-repeat: no-repeat;
+    
+
+}
+.flex_text1{
+    display: flex;
+    flex-direction: column;
+    align-items:center;
+    justify-content: flex-start;
+    margin-left: 280px;
+    padding-top: 100px;
+    font-size: 130%;
+    
+
+}
+@media ( max-width:800px){
+    .flex_text1{
+        display: flex;
+        margin-left: 50%;
+        padding-top: 0;
+        justify-content: center;
+}}
+.left1{
+    position: absolute; 
+    color:orange;
+    width: 400px;
+    background-color:rgba(29, 28, 28, 0.884);
+}
+
+.left2{
+    position: absolute; 
+    color:black;
+    background-color:orange;
+    width: 400px;
+    margin-top: 100px;
+}
+
+@media ( max-width:800px){
+    .left1{
+        width:250px;
+        font-size: 95%;
+    }
+    .left2{
+        width: 250px;
+        font-size: 95%;
+        
+
+}}
+
+
+/* //////////////////// */
+.container_lorem{
+    display: flex;
+
+}
+.col_1{
+    color:orange;
+    /* justify-content: end; */
+    font-style: italic;
+}
+.lorem_collum{
+    width:27%;
+    margin: 8px;
+}
+
+/* //////////////////////// */
+.orange_text{
+    color: orange;
+    font-weight: bolder;
+}
+.black_text{
+    font-weight: bolder;
+}
+.rabbit_img{
+    display: flex;
+    margin: 2%;
+
+
+}
+
+ @media ( max-width:800px){
+    .rabbit_img{
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+     
+     
+    }}
+    
+
+
+.span_center{
+    display: flex;
+    justify-content: center;
+    
+    
+}
+@media (max-width: 800px){
+
+    .span_center{
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;    
+    }
+}
+/* .rabbit1.rabbit2.rabbit3 {
+    width: 320px;
+    height: 320px;
+    cursor: pointer;
+    background-color: orange;
+    border-radius: 220px;
+    border:solid orange;
+    background-size: cover;
+    background-repeat: no-repeat;
+
+   } */
+   .rabbit1{
```

Comment:
All rabbits almost the same. Create one class for all rabbits

Reply:
I tried this way and through (* and ^), but it did not work. I'm sorry